### PR TITLE
[MI-3337]:Fixed review comment of PR mattermost#961 and added TC

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**
https://github.com/mattermost/mattermost-plugin-jira/pull/961

### CODE REVIEW

File: server/testdata/webhook-server-issue-updated-commented-mention-user.json [hunk 1]
```diff
@@ -0,0 +1,606 @@
...
+            "key": "levbrouk",
+            "emailAddress": "lev@mattermost.com",
+            "avatarUrls": {
+                "48x48": "https://www.gravatar.com/avatar/8c504b6c8f13cf9dc9ffb8af57323320?d=mm&s=48",
+                "24x24": "https://www.gravatar.com/avatar/8c504b6c8f13cf9dc9ffb8af57323320?d=mm&s=24",
+                "16x16": "https://www.gravatar.com/avatar/8c504b6c8f13cf9dc9ffb8af57323320?d=mm&s=16",
+                "32x32": "https://www.gravatar.com/avatar/8c504b6c8f13cf9dc9ffb8af57323320?d=mm&s=32"
+            },
+            "displayName": "Lev Brouk",
+            "active": true,
```
Comment (manojmalik20) [inline server/testdata/webhook-server-issue-updated-commented-mention-user.json:584]: Please ensure that all this test data does not contain any real sensitive or confidential data.

File: server/webhook_http_test.go [hunk 2]
```diff
@@ -600,6 +600,20 @@ func TestWebhookHTTP(t *testing.T) {
...
+			ExpectedSlackAttachment: true,
+			ExpectedHeadline:        "Test User **commented** on story [TES-41: Unit test summary 1](https://some-instance-test.atlassian.net/browse/TES-41)",
+			ExpectedText:            "> Added a comment with mentioned user [~test-mm-username]",
+			CurrentInstance:         false,
+		},
+		"SERVER issue commented - user mentioned": {
+			Request:                 testWebhookRequest("webhook-server-issue-updated-commented-mention-user.json"),
+			ExpectedSlackAttachment: true,
+			ExpectedHeadline:        "Lev Brouk **commented** on story [PRJX-14: As a user, I can find important items on the board by using the customisable ...](http://sales-jira.centralus.cloudapp.azure.com:8080/browse/PRJX-14)",
+			ExpectedText:            "> unik with mentioned user [~test-mm-username] ",
```
Comment (manojmalik20) [inline server/webhook_http_test.go:614]: What is `unik` here?

**Review by manojmalik20 — APPROVED** [review]

File: server/testdata/webhook-server-issue-updated-commented-mention-user.json (hunk 1 above)
Comment (Kshitij-Katiyar) [reply server/testdata/webhook-server-issue-updated-commented-mention-user.json:584]: copied from same already present file.

File: server/webhook_http_test.go (hunk 2 above)
Comment (Kshitij-Katiyar) [reply server/webhook_http_test.go:614]: copied from same already present file.

**Pull request merged**